be/src/vec/olap/vgeneric_iterators.h

```diff
@@ -120,6 +120,21 @@ class VMergeIteratorContext {
 
     void set_skip(bool skip) const { _skip = skip; }
 
+    bool is_same() const { return _same; }
+
+    void set_same(bool same) const { _same = same; }
+
+    std::vector<bool> get_pre_ctx_same() const { return _pre_ctx_same_bit; }
```

Comment:
return a const reference to avoid copy of the whole vector.

Reply:
ok, i fix it